include/caf/events/led_event.h

```diff
@@ -20,22 +20,40 @@
 extern "C" {
```

Comment:
For the defgroup in lines 10-14, please follow the guidelines here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#file-headers-and-groups

(Global edit for all files)

Reply:
fixed